schema/meltano.schema.json

```diff
@@ -27,6 +27,67 @@
       "description": "A database URI for the project system database. Defaults to a SQLite file stored at .meltano/meltano.db",
       "default": "sqlite:///$MELTANO_PROJECT_ROOT/.meltano/meltano.db"
     },
+      "state_backend": {
+        "type": "object",
+          "description": "Configuration for state backend.",
+          "properties": {
+            "uri": {
+              "type": "string",
+              "description": "The URI pointing to the backend to use.",
+              "default": "systemdb"
+              },
+            "lock_timeout_seconds": {
+              "type": "integer",
+              "description": "The number of seconds a lock on a state file is considered valid."
+              },
+            "lock_retry_seconds": {
+              "type": "integer",
+              "description": "The number of seconds to wait between retrying lock acquisition."
+              },
+            "az": {
```

Comment:
I haven't seen "az" used as an acronym for Azure Blob Storage before. Is there precedence for this? `azure` is used elsewhere in this PR for this setting.

Reply:
I think there's some precedence in Azure's docs, but agreed that it should be `azure` for our purposes. Updated.